bootstrap.py

```diff
@@ -689,6 +703,9 @@ class MEOptions:
 
 if __name__ == '__main__':
 
+    # Check if Satellite or Capsule are installed or not
+    check_prerequisite()
```

Comment:
This should probably go somewhere around https://github.com/peppos/katello-client-bootstrap/blob/6198337c0b1172d90d238d0b009e3fb9883212de/bootstrap.py#L880

(either before or after the `root` check).